client/src/plugins/editor-events/EditorEvents.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import { PureComponent } from 'react';
+
+import BpmnModelerExtension from './bpmn-modeler-extension';
+
+
+/**
+ * An extension that shows how to hook into
+ * editor events to accomplish the following:
+ *
+ * - hook into <bpmn.modeler.configure> to provide a bpmn.modeler extension
+ * - hook into <bpmn.modeler.created> to register for bpmn.modeler events
+ * - hook into <tab.saved> to perform a post-safe action
+ *
+ */
+export default class EditorEvents extends PureComponent {
+
+  constructor(props) {
+
+    super(props);
+
+    const {
+      subscribe
+    } = props;
+
+    subscribe('bpmn.modeler.configure', (event) => {
```

Comment:
This doesn't work (yet) since no `bpmn.modeler.configure` event is emitted by the app. Are we okay with that?

Reply:
I'll add the remaining issues for it and move this example to the test plug-in section.